backend/.env

```
@@ -1,14 +1,26 @@
-DATABASE_URL="postgresql://matrimonial_user:matrimonial123@localhost:5432/matrimonial_db"
-JWT_SECRET="your-super-secret-jwt-key-change-in-production"
-REDIS_URL="redis://localhost:6379"
-AWS_ACCESS_KEY_ID="your-aws-access-key"
-AWS_SECRET_ACCESS_KEY="your-aws-secret-key"
-AWS_REGION="us-east-1"
-S3_BUCKET_NAME="matrimonial-uploads"
+DATABASE_URL="postgresql://postgres:Shoaib%40802409@127.0.0.1:5432/matrimonial_db"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)